tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -72,13 +55,6 @@ setup() {
 
     add_allow_all_policy_to_yaml "$kata_pod_with_nydus_config"
     k8s_create_pod "$kata_pod_with_nydus_config"
-    echo "Kata pod test-e2e with nydus annotation is running"
-
-    echo "Checking the image was pulled in the guest"
```

Comment:
Hi @ChengyuZhu6 !

I didn't understand why you removed this check. How do we ensure it was really pulled by the guest?

Reply:
Hi @wainersm! In the coco CI job, we're setting `shared_fs` to `none`, as seen in the Makefile: https://github.com/kata-containers/kata-containers/blob/91084058ae620d7c3c2053cdac2de0445824fbc2/src/runtime/Makefile#L248-L252. This change implies that all kubernetes tests will now pull images in the guest. Consequently, there's no longer a need to check.

Reply:
My understanding of `shared_fs=none` is that even not mounting filesystems, it will be copying files over the guest (e.g. config maps). I don't know if it will attempt to copy images...

Anyway, afaik we don't have any test to test `shared_fs=none` really works. Having that checking on this test case will ensure that images are pulling from the guest as intended. Is it too much work keep this checking?